script.js

```diff
@@ -0,0 +1,61 @@
+let buttonCalculate = document.querySelector(".btn");
```

Comment:
Since you are not reassigning the button element, `const` is more appropriate than `let`. 
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

```suggestion
const buttonCalculate = document.querySelector(".btn");
```

This suggestion also applies to all other `let` keywords in this file